tests/runc/main.yml

```diff
@@ -0,0 +1,135 @@
+---
+# vim: set ft=ansible:
+#
+# !!!NOTE!!! This playbook was tested using Ansible 2.1; it is recommended
+# that the same version is used.
+#
+# This playbook is a basic functional test for runc.  It configures the
+# host with the required packages and libraries to run the runc upstream
+# tests found at https://fedorapeople.org/cgit/caiqian/public_git/runctst.git
+#
+#  - runc create
+#  - runc delete
+#  - runc exec
+#  - runc init
+#  - runc kill
+#  - runc list
+#  - runc ps
+#  - runc run
+#  - runc start
+#  - runc state
+#  - runc update
+#
+
+- name: runc - Setup
+  hosts: all
+  become: yes
+
+  tags:
+    - setup
+
+  vars_files:
+    - vars.yml
+
+  roles:
+    - role: runc_version_check
+      runc_version: "{{ g_runc_version }}"
+      tags:
+        - runc_version_check
+
+    # Subscribe if the system is RHEL
+    - role: redhat_subscription
+      when: ansible_distribution == 'RedHat'
+      tags:
+        - redhat_subscription
+
+    - role: rpm_ostree_install
+      packages: 'gcc python-devel'
+      reboot: false
+      when: ansible_distribution == 'RedHat' or "'CentOS' in ansible_distribution"
+      tags:
+        - rpm_ostree_install
+
+    # Waiting on https://bodhi.fedoraproject.org/updates/FEDORA-2017-6319e78c81
+    #  to fix criu dependency
+
+    - role: rpm_ostree_install
+      packages: 'gcc python-devel redhat-rpm-config criu'
+      reboot: false
+      when: ansible_distribution == 'Fedora'
+      tags:
+        - rpm_ostree_install_fedora
+
+    - role: reboot
+      tags:
+        - reboot
+
+  post_tasks:
+    - name: Turn on user namespaces
+      shell: grubby --args="user_namespace.enable=1" --update-kernel="$(grubby --default-kernel)"
+      when: ("'CentOS' in ansible_distribution" or ansible_distribution == 'Redhat')
+
+    - include: './roles/reboot/tasks/main.yml'
```

Comment:
There are a lot of relative paths from the includes and roles with includes which, combined with the directory structure, makes Ansible confused.  For this particular instance, I should just include common/ans_reboot.yaml but we need to find a cleaner way of doing the includes.